akka-persistence-testkit/src/main/scala/akka/persistence/testkit/internal/EventSourcedBehaviorTestKitImpl.scala

```diff
@@ -150,10 +132,74 @@ import akka.persistence.typed.internal.EventSourcedBehaviorImpl
       replyProbe.stop()
     }
 
+    val newState = getState()
+    val newEvents = getEvents(oldEvents.size)
+
+    postCommandCheck(newEvents, newState, Some(reply))
+
+    CommandResultImpl[Command, Event, State, R](command, newEvents, newState, Some(reply))
+  }
+
+  override def runCommandExpectNoReply[R](creator: ActorRef[R] => Command): CommandResult[Command, Event, State] = {
+    val replyProbe = actorTestKit.createTestProbe[R]()
+    val command = creator(replyProbe.ref)
+    preCommandCheck(command)
+    val oldEvents = getEvents(dropOldEvents = 0)
+
+    actor ! command
+
+    try {
+      replyProbe.expectNoMessage()
+    } catch {
+      case NonFatal(e) =>
+        throw new AssertionError(s"Received unexpected reply for command [$command]. ${e.getMessage}")
+    } finally {
+      replyProbe.stop()
+    }
+
+    val newState = getState()
+    val newEvents = getEvents(oldEvents.size)
+
+    postCommandCheck(newEvents, newState, reply = None)
+
+    CommandResultImpl[Command, Event, State, Nothing](command, newEvents, newState, None)
+  }
+
+  private def getEvents[R](dropOldEvents: Int): immutable.Seq[Event] = {
+    // FIXME we can expand the api of persistenceTestKit to read from storage from a seqNr instead
+    persistenceTestKit.persistedInStorage(persistenceId.id).map(_.asInstanceOf[Event]).drop(dropOldEvents)
+  }
+
+  private def getState(): State = {
+    internalActor ! EventSourcedBehaviorImpl.GetState(stateProbe.ref)
+    val newState = stateProbe.receiveMessage()
+    newState
+  }
```

Comment:
Feature request: 
Can you expose this to the test kit api so it would be possible to query for the state from a test without having to run a command to do so?

Reply:
Sure, I will.